### PR TITLE
POWR-4057 migrate pccep meetings

### PR DESCRIPTION
**Pull request by ccdatapdx**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -240,7 +240,8 @@
             },
             "drupal/group": {
                 "2842630 - Empty page when trying to create group node": "https://www.drupal.org/files/issues/2019-01-04/2842630-20.patch",
-                "3020883 - Use VBO together with group permission": "https://www.drupal.org/files/issues/2020-08-17/3020883-22.patch"
+                "3020883 - Use VBO together with group permission": "https://www.drupal.org/files/issues/2020-08-17/3020883-22.patch",
+                "3268711 - Error: Call to a member function label() on null in line 131 of modules/contrib/group/src/Entity/Controller/GroupListBuilder.php).": "https://www.drupal.org/files/issues/2022-03-09/Group-3268711-1.patch"
```
Comment (joshuami) [inline composer.json:244]: I'm not seeing the error this patch refers to on any environment. The issue states that the patch is for Drupal 9.3 and Groups 1.4. We are using Drupal 9.2 still on portlandor. Please remove this patch or explain its use.

**Review by joshuami — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (ccdatapdx) [reply composer.json:244]: The error was quite similar to the one on the issue, and was produced on my local: 
`Error: Call to a member function label() on null in Drupal\group\Entity\Controller\GroupListBuilder->buildRow() (line 131 of modules/contrib/group/src/Entity/Controller/GroupListBuilder.php).`

How would I go about producing this error on an environment? Do you mean producing it on a multidev?

File: web/modules/custom/portland_migrations/migrations/pccep_meetings.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    plugin: format_date
+    from_format: 'Y-m-d H:i:s T'
+    to_format: 'Y-m-d\TH:i:s'
+    source: LAST_UPDATED_DATE
+  field_display_groups:
+      plugin: default_value
+      default_value: 419
+destination:
+  plugin: 'entity:node'
+  default_bundle: event
```
[the remark this answers is not in the document]
Comment (ccdatapdx) [reply web/modules/custom/portland_migrations/migrations/pccep_meetings.yml:86]: corrected in fresh branch

File: web/modules/custom/portland_migrations/migrations/pccep_meetings_redirects.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    plugin: str_replace
+    case_insensitivity: true
+    source: URL
+    search: https://www.portlandoregon.gov/
+    replace: ''
+  redirect_redirect:
+    plugin: concat
+    source:
+      - constants/redirect_prefix
+      - "@temp_nid"
```
[the remark this answers is not in the document]
Comment (ccdatapdx) [reply web/modules/custom/portland_migrations/migrations/pccep_meetings_redirects.yml:34]: corrected in fresh branch

File: composer.json (hunk 1 above)
Comment (ccdatapdx) [reply composer.json:244]: Good news, the fresh branch didn't need the patch

**Comment by ccdatapdx** [comment]
Has a second branch

**Pull request closed without merging**